pkgs/prost/processor.go

```diff
@@ -63,14 +63,25 @@ func (ep *EventProcessor) MonitorEvents() {
 
```

Comment:
For the first run with `lastProcessedBlock=0` , the `for` loop will begin iterating from `targetBlockNumber+1` because `lastProcessedBlock = targetBlockNumber`, and how will the conditional of `blockNum <= targetBlockNumber` be satisfied?

@Seth-Schmidt